pages/team.md

```diff
@@ -10,7 +10,7 @@ We also invite community members to reach out to [mdanalysis@numfocus.org](mailt
 
 ## Roles
 
-In this section we list major contributors who are currently contributing substantial effort to the roles listed below. This team organisation structure was started in November 2023; many people contributed to these tasks before then and may not appear in this table, in particular the [emeritus core developers]({{ site.url }}{% link pages/about.md %}#mdanalysis-emeriti-core-developers).
+In this section we list major contributors who are currently contributing substantial effort to the roles listed below. This team organisation structure was started in November 2023; many people contributed to these tasks before then and may not appear in this table, in particular the [emeritus core developers]({{ site.url }}{% link about.md %}#mdanalysis-emeriti-core-developers).
```

Comment:
"major contributors who are currently devoting" ... (sorry this is extremely minor)